app/javascript/components/widgets/widgets/forest-change/tree-loss-ranked/selectors.js

```diff
@@ -108,17 +108,26 @@ export const parseData = createSelector(
       }
       dataTrimmed = dataTrimmed.slice(trimStart, trimEnd);
     }
-    return dataTrimmed.map(d => ({
-      ...d,
-      color: colors.main,
-      path: getAdminPath({
-        ...location,
-        country: location.region && location.country,
-        query,
-        id: d.id
-      }),
-      value: settings.unit === 'ha' ? d.loss : d.percentage
-    }));
+    return dataTrimmed.map(d => {
+      let value;
+      if (settings.unit === 'ha') {
+        value = d.loss;
+      } else if (d.percentage <= 100) {
```

Comment:
This doesn't fix the issue of the percentages being greater than 100, it just limits them. We need to make sure the percentage calculation is correct in the first place.